README.md

```diff
@@ -31,6 +31,15 @@ find research data from the RKI.
 For further details, please consult our
 [project page](https://www.rki.de/DE/Content/Forsch/MEx/MEx_node.html).
 
+**Contact** \
+For more information, please feel free to email us at [mex@rki.de](mailto:mex@rki.de).
+
+### Publisher of this document
+**Robert Koch-Institut** \
+Nordufer 20 \
+13353 Berlin \
+Germany
+
```

Comment:
if you want this in here, it should go into the template to be propagated into all repos https://github.com/robert-koch-institut/mex-template/blob/0.3.0/mex-%7B%7B%20cookiecutter.project_name%20%7D%7D/README.md?plain=1#L38

Reply:
I think the information would be nice to have, and maybe also good to have the official RKI address present as well (kind of having an "impressum"). what do you think? if you agrre, then I can put this into mex-template.